benchmark/run.rb

```diff
@@ -19,6 +19,9 @@ module GraphQLBenchmark
   BIG_SCHEMA = GraphQL::Schema.from_definition(File.join(BENCHMARK_PATH, "big_schema.graphql"))
   BIG_QUERY = GraphQL.parse(File.read(File.join(BENCHMARK_PATH, "big_query.graphql")))
 
+  FIELDS_WILL_MERGE_SCHEMA = GraphQL::Schema.from_definition(File.join(BENCHMARK_PATH, "fields_will_merge_schema.graphql"))
+  FIELDS_WILL_MERGE_QUERY = GraphQL.parse(File.read(File.join(BENCHMARK_PATH, "fields_will_merge_query.graphql")))
```

Comment:
I think it would increase the clarity of this query's intent and simplify maintenance if this string was created dynamically instead of dumped to a file. For example: 

```suggestion
  FIELDS_WILL_MERGE_QUERY = GraphQL.parse("{ #{Array.new(5000, "hello").join(" ")} }")
```

(That makes it immediately clear that "this query string is 'hello' 5000 times in a row"!)